src/robot/libdocpkg/model.py

```diff
@@ -137,6 +147,10 @@ def deprecated(self):
     def _sort_key(self):
         return self.name.lower()
 
+    def _generate_shortdoc(self):
+        if not self._shortdoc:
+            self.shortdoc = self._get_shortdoc()
```

Comment:
This doesn't seem to be used anywhere.

Reply:
It is used. It was just by accident a private method.

That is really important for generating the shortdoc when the keywords list is set to LibraryDoc object.
and it is only generated if the shortdoc is not already set, what means it only happens in Java- and robotbbuilder, but not in json- and xmlbuilder.